Orange/widgets/data/owpythonscript.py

```diff
@@ -645,7 +667,7 @@ def saveScript(self):
             f.close()
 
     def initial_locals_state(self):
-        d = dict([(i.name, getattr(self, i.name, None)) for i in self.inputs])
+        d = dict([(name, getattr(self, name, None)) for name in self.inputs_names])
```

Comment:
I completely forgot about this. Well, lists cannot be passed. There needs to be a dictionary since some other code (`class InteractiveInterpreter`) does not accept lists.